src/app/features/courses/services/course.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+
+import { Course } from '../course.model';
+import { courses } from './courses.mock';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
If this service you use just in 'courses' module be better to import this one into current module instead of 'root' provider.